test/functional/helpers.lua

```diff
@@ -25,6 +25,7 @@ local write_file = global_helpers.write_file
 local module = {
   NIL = mpack.NIL,
   mkdir = lfs.mkdir,
+  currentdir = lfs.currentdir,
```

Comment:
you can just `require('lfs')` in the test directly. We don't generally add such things to `helpers.lua`.